modules/bvl_feedback/php/NDB_Form_bvl_feedback.class.inc

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: evanmcilroy
+ * Date: 15-06-05
+ * Time: 12:52 PM
+ */
+
+require_once "NDB_Form.class.inc";
+
+class NDB_Form_bvl_feedback extends NDB_Form
+{
+    function _hasAccess()
+    {
+        // create user object
+        $user =& User::singleton();
+        if (Utility::isErrorX($user)) {
+            return PEAR::raiseError("User Error: " . $user->getMessage());
+        }
+
+        $site =& Site::singleton($user->getData('CenterID'));
+        if (Utility::isErrorX($site)) {
+            return PEAR::raiseError("Unable to construct the list_of_sites array: " . $site->getMessage());
+        }
```

Comment:
No need for PHP4 style exceptions, real exceptions should be used now.. you can just get rid of the isErrorX call.